### PR TITLE
Base

### PR DESCRIPTION
**Pull request by AymanJabr**
Styled the Html page with 3 separate styles for different screens. 
One for a phone with max width: 768px
One for tablets with max width: 1024px
One for pc with bigger screens

### CODE REVIEW

File: styles.css
Comment (MuriloRoque) [inline styles.css]: - [ ] In here you have two options to make your code cleaner:
- a) Make a class named `d-none` or similar with `display: none` and apply it to these elements you want or;
- b) Put the class names in the same line separated with a comma, then apply `display: none` to them at the same time.

File: styles.css
Comment (MuriloRoque) [inline styles.css]: - [ ] In here you have two options to make your code cleaner:
- a) Make a class named `f-17` or similar with `font-size: 17px` and apply it to these elements you want or;
- b) Put the class names in the same line separated with a comma, then apply `font-size: 17px` to them at the same time.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @AymanJabr and @vidhishweta01 :wave: 

### You did such an amazing job on this project, I'm very proud of you! Congratulations :clap: :clap: :clap: 

### There are only some very minor issues to solve, but they are intended to improve your code and make it cleaner. Please check my comments in your code.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request Changes
Please remove linter errors and then request for a code review.

Best,
Arslan Bisharat
Slack:arslanbisharat

**Comment by AymanJabr** [comment]
Linter errors are from netlify and not from the Microverse linter.

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
